src/content-blocks/LeadersBlock/index.tsx

```diff
@@ -0,0 +1,23 @@
+import BlockContainer from '@/components/atoms/BlockContainer';
+import { LeadersBLock as LeadersBlockProps } from './type';
+import Leaders from '@/components/organisms/Leaders';
+
+type Props = LeadersBlockProps & React.HTMLAttributes<HTMLElement>;
+
+const LeadersBlock = ({ title, leaders, cta, blockProperties }: Props) => {
+  return (
+    <BlockContainer
+      variant={blockProperties.variant}
+      orientation={blockProperties.orientation}
+      slug={blockProperties.slug}
+      cta={cta}
+    >
+      <section className="sl-layout">
```

Comment:
De `BlockContainer` is zelf al een `section`. Het is dus niet nodig om hier ook nog een `section` in te steken.